app/views/area/faqs/index.html.erb

```diff
@@ -5,7 +5,7 @@
 
     <div class="faq-content">
         <div class="container">
-            <a href="/<%= @watched_area.slug %>"><i class="fas fa-arrow-left"></i> Back</a>
+            <a href="/<%= @watched_area.slug %>"><i class="fas fa-arrow-left"></i><button type="button" class= "btn btn-outline-dark"> Back</button></a>
```

Comment:
We can get rid of `<i class="fas fa-arrow-left"></i>` here because I don't believe this actually renders anything on the page. `fas` was an old icon library I used but have since removed from the project